test/unit/specs/Calendar.spec.js

```diff
@@ -0,0 +1,48 @@
+import Vue from 'vue'
+import Calendar from '@/components/Calendar'
+
+describe('Calendar.vue', function () {
+  const Constructor = Vue.extend(Calendar)
+  const vm = new Constructor().$mount()
+
+  // Following test is for method "addOneday".
+  // There are three test cases in total: day change, month change and year change
+  it('Test addOneday Method', (done) => {
+    // Fake Slide
+    const fakeSlide = {
+      date: {
+        content: '2018-01-01'
+      }
+    }
+    // To check whether the one day is added correctly
+    // Should be "2018-01-02"
+    expect((vm.addOneDay(fakeSlide)).toDateString)
+      .to.equal(('2018-01-02').toDateString)
+
+    // Now testing whether the month can be changed correctly.
+    // Change the date to '2018-01-31'
+    fakeSlide.date.content = '2018-01-31'
```

Comment:
Should this be using `$data`?

Reply:
No……there's no "Data" function in the original calendar view at all. The data here is just created by this test component so $data is not necessary here

Reply:
Thanks!

Reply:
You are welcome!